modules/Hoot/ui/sidebar/layerConflate.js

```diff
@@ -300,6 +300,13 @@ class LayerConflate extends SidebarForm {
             this.advancedOptions.toggle();
         }
 
+        if (this.newFolderNameInput.node().value.length > 0) {
+            let newFolderName = this.newFolderNameInput.node().value;
+            let newFolderId;
+            newFolderId = (await Hoot.folders.addFolder( newFolderName, newFolderName )).folderId;
```

Comment:
I think if we want the new folder to always be created at the root, then the first param to `addFolder` should be empty string.  This sort of works if the `newFolderName` doesn't exist, but would get weird if the folder name did exist and would create another folder with the same name underneath.

It might be a little confusing that the `Path` drop down doesn't allow a root ` / ` folder to be shown, even though the new folder will be created at the root.  Maybe the placeholder for the `Path` drop down should be ` / `, but if not creating a new folder it should require the user to make a choice of an existing folder.